.husky/pre-commit

```diff
@@ -8,6 +8,7 @@ IGNORED_FILES=(
   ".snippets/code/develop/toolkit/parachains/spawn-chains/zombienet/write-tests/big-network-test.toml"
   ".snippets/code/develop/toolkit/parachains/spawn-chains/zombienet/write-tests/small-network-test.toml"
   ".snippets/code/develop/toolkit/parachains/spawn-chains/zombienet/write-tests/spawn-a-basic-chain-test.toml"
+  ".snippets/code/tutorials/polkadot-sdk/parachains/build-custom-pallet/pallet-unit-testing/cargo-dev-dependencies.toml"
```

Comment:
@eshaben I intentionally added this file here because we want to keep the "..." in some lines of that file to show that there is content before and after it